app/src/main/java/org/mozilla/reference/browser/NotificationManager.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.mozilla.reference.browser
+
+import android.annotation.TargetApi
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager as AndroidNotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Build
+import android.os.Build.VERSION.SDK_INT
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.core.content.ContextCompat
+import mozilla.components.concept.sync.Device
+import mozilla.components.concept.sync.TabData
+import mozilla.components.support.base.log.logger.Logger
+
+/**
+ * Manages notification channels and allows displaying different types of notifications.
+ */
+class NotificationManager(private val context: Context) {
+    companion object {
+        const val RECEIVE_TABS_TAG = "ReceivedTabs"
+        const val RECEIVE_TABS_CHANNEL_ID = "ReceivedTabsChannel"
```

Comment:
In other components we started to prefix this with `mozac.feature.foo.` too in order to avoid conflicts.

Reply:
Prefixed with `org.mozilla.reference.browser.` since this is an r-b class.